manifest.json

```
@@ -3,7 +3,7 @@
   "id": "shelly-adapter",
   "name": "Shelly",
   "short_name": "Shelly",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Can you please reset this to
```suggestion
  "version": "0.6.0",
```
I'll bump the version in the release commit.
It will be `0.7.0` because you added a new feature.

---

ok

---

ok